keps/sig-multicluster/4322-cluster-inventory/README.md

```diff
@@ -467,6 +471,32 @@ to be added by the cluster manager upon creation. The value of the label
 MUST be the same as the name of the cluster manager. The purpose of this
 label is to make filter clusters from different cluster managers easier.
 
+#### Endpoints and access credentials
+
+Three fields provide the information needed to access the cluster's API endpoint.
+
+##### Endpoints
+
+A list of API endpoints. Having a list allows endpoints to roll over:
+when an endpoint needs to change, the new endpoint is added to the list,
+and when the old endpoint stops being available, it is removed. Clients
+need to try all listed endpoints until they find one which they can
+connect to.
+
+##### Provider
+
+The name of a credential provider. This is a value set by the cluster
+manager identifying how the credential information is supposed to be
+used.
+
+##### Credential object reference
```

Comment:
sorry I missed kubecon where this may have been discussed more, but would this object contain an actual credential? what would this credential be used by and for?

It seems to me that multicluster controllers that want to leverage the ClusterProfile should provide their own credentials as they may have different needs on the cluster itself and therefore different authorization mapped to them.